relayer/processor/path_processor.go

```diff
@@ -207,6 +207,8 @@ type PathEndMessages struct {
 	MsgTransfer        PacketSequenceCache
 	MsgRecvPacket      PacketSequenceCache
 	MsgAcknowledgement PacketSequenceCache
+	MsgTimeout         PacketSequenceCache
+	// TODO MsgTimeoutOnClose
```

Comment:
This TODO is valid?

Reply:
It was, since the relayer does not yet ever send `MsgTimeoutOnClose`, but This is actually dead code now after cleanup in prior PRs. Removed.